test/jdk/javax/swing/JTableHeader/TableHeaderRendererTest.java

```diff
@@ -29,25 +29,31 @@
 import javax.swing.JTable;
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
 import javax.swing.table.AbstractTableModel;
 import javax.swing.table.JTableHeader;
 import javax.swing.table.TableCellRenderer;
 
 /*
  * @test
  * @bug 6429812
- * @requires (os.family == "windows")
  * @summary Test to check if table is printed without NPE
  * @run main TableHeaderRendererTest
  */
 
 public class TableHeaderRendererTest {
+    static String LnFName = null;
     public static void main(String[] args) throws Exception {
-        UIManager.setLookAndFeel("com.sun.java.swing.plaf.windows.WindowsLookAndFeel");
-
-        // initialize should not throw NullPointerException
-        SwingUtilities.invokeAndWait(TableHeaderRendererTest::initialize);
 
+        UIManager.LookAndFeelInfo[] lookAndFeel = UIManager.getInstalledLookAndFeels();
+        for (UIManager.LookAndFeelInfo look : lookAndFeel) {
+            // initialize should not throw NullPointerException
+            LnFName = look.getName();
```

Comment:
Why not declare it here? It does not need to be a field, moreover you don't need it even as a local variable.